### ISSUE TITLE
Reply button disappear on click in PR when language is spanish

### ISSUE THREAD

**Issue by tonivj5** [go-gitea/gitea#5237]
- Gitea version (or commit ref): 7edb930
- Git version: 2.1.4
- Operating system: Debian
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (provide example URL): https://try.gitea.io/xxxtonixxx/asdf/pulls/1#issuecomment-8383
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description
I think it's self-explanatory

## Screenshots
- Pre-clicked
![image](https://user-images.githubusercontent.com/7110786/47813857-82696b80-dd4c-11e8-91a7-66c8eb2a0eee.png)
- Post-clicked
![image](https://user-images.githubusercontent.com/7110786/47813873-8d240080-dd4c-11e8-937d-b20f1002aa23.png)
- In another languages (Deutsch). It works
![image](https://user-images.githubusercontent.com/7110786/47813899-9ca34980-dd4c-11e8-94bf-8a6fb455dcfe.png)
- However, in files changed I can click and reply :laughing:  
![image](https://user-images.githubusercontent.com/7110786/47813943-b5abfa80-dd4c-11e8-92e7-3f4a36217fca.png)

**Comment by Caballerog** [comment]
I have the same problem using the last docker image. I've used the english language which works.

**Comment by zeripath** [comment]
OK I think that the locale-es_ES.ini file has a problem. There are likely other problems in that file too.

line 459: 
```ini
...
issues.commented_at=`comentado <a href="#%s"> %s`</a>
...
```

is the cause of this issue. The backtick (\`) before the `</a>` means that the `<a href="...">` is never closed.

**Comment by techknowlogick** [comment]
@zeripath thanks for digging deep into this issue. I was worried that it would've been something like that because why would it work for english and not other languages. This means that the linked PR won't work, and the fixes will need to be put into crowdin.

cc: @lafriks

**Comment by techknowlogick** [comment]
Oh wait. PR still may work.

**Comment by lafriks** [comment]
I have fixed Spanish translation in crowdin but #5239 fixes other problem I think
